### PR TITLE
Branch safe delete

### PR DESCRIPTION
**Pull request by cheekean5848**

### CODE REVIEW

File: src/test/java/seedu/address/testutil/TypicalPersons.java [hunk 1]
```diff
@@ -76,7 +76,7 @@ private TypicalPersons() {} // prevents instantiation
     public static AddressBook getTypicalAddressBook() {
```
Comment (clement0010) [inline src/test/java/seedu/address/testutil/TypicalPersons.java:76]: Small thing but AddressBook would also need to be updated

**Review by clement0010 — APPROVED** [review]
LGTM! Thanks for refactoring `person` to `patient`

File: src/test/java/seedu/address/testutil/TypicalPersons.java (hunk 1 above)
Comment (cheekean5848) [reply src/test/java/seedu/address/testutil/TypicalPersons.java:76]: update to MedBook?

Comment (clement0010) [reply src/test/java/seedu/address/testutil/TypicalPersons.java:76]: yes

**Pull request merged**